variables.tf

```diff
@@ -108,4 +108,10 @@ variable "redis_connection_string" {
   type        = string
   description = "The redis connection string."
   default     = ""
+}
+
+variable "publish_custom_metrics" {
+  type        = string
+  description = "Publish custom metrics to monitor Weights & Biases"
+  default     = ""
```

Comment:
Which custom metrics? Does it default to on or off (what are acceptable values since it is a string)? Could we add more documentation here for the consumer?

Reply:
my bad, I should have left more of a description with the pr, it's a bit vague. My goal is to pass a string here to indicate to gorilla to publish reliability metrics (right now it is just the queue size). The string can be "s3://" for aws and in the future "gs://" for gcp. It is for monitoring purposes and more relevant to our team for managing single-tenant instances. It's not really something I think consumers outside of us will be interested in. If you have some other suggestions as to how to the best way to go about this, I'm all ears.

Reply:
The default is off

Reply:
@zythosec I made a few updates, any other thoughts on this topic?